features/smoke-tests/buyer/direct-award.feature

```diff
@@ -0,0 +1,10 @@
+@skip-staging @skip-production
+@smoke-tests
+@direct-award-passive
+Feature: Direct Award passive assurance
+
+Scenario: User can save a search into a new Direct Award Project
```

Comment:
I think I'd call this "User is invited to authenticate in order to safe a Direct Award search" to distinguish it from the logged-in version? (or do we always get really context of the 'feature' title, in which case maybe just clarifying that would be enough - "passive assurance" isn't so clear to me).